src/main/java/jenkins/plugins/gerrit/client/ssh/GerritClientSSH.java

```diff
@@ -0,0 +1,188 @@
+package jenkins.plugins.gerrit.client.ssh;
+
+import com.google.common.base.Ascii;
+import com.google.gerrit.extensions.api.changes.*;
+import com.google.gerrit.extensions.common.ChangeInfo;
+import com.google.gerrit.extensions.restapi.RestApiException;
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.jcraft.jsch.Session;
+import com.urswolfer.gerrit.client.rest.http.changes.ChangesParser;
+import com.urswolfer.gerrit.client.rest.http.changes.ReviewResultParser;
+import jenkins.plugins.gerrit.client.GerritClient;
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang.StringUtils;
+import org.eclipse.jgit.transport.RemoteSession;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.Charset;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class GerritClientSSH implements GerritClient {
+
+    private final int timeout;
+    private final Logger logger = LoggerFactory.getLogger(GerritClientSSH.class);

```

Comment:
please use the logger of jenkins it is available at the context listener.

Reply:
ok!

Reply:
By the way, isn't the logger available in the context listener more for return input to the end-user? I.e. not suitable for debug-logging.